common.js

```diff
@@ -28,10 +28,17 @@ const initTiny = (nm, rndcls, attrs) => `
         $('textarea#input${text(nm)}_${rndcls}').trigger('change');
       } 
       let changeDebounced = $.debounce ? $.debounce(tmceOnChange, 500, null,true) : tmceOnChange;
+      window.tinymce.PluginManager.add('drawio', getDrawioPlugin(${
+        typeof attrs?.folder === "string" &&
+        attrs.folder !== "Base64 encode" &&
+        attrs.folder !== ""
+      }, ${attrs?.min_role_read || public_user_role}));
       const ed = await tinymce.init({
+        extended_valid_elements: 'div[*],img[*]',
+        valid_children: ['+div[img]'],
         selector: '.${rndcls}',
         promotion: false,
-        plugins: ['link', 'fullscreen', 'charmap', 'table', 'lists', 'searchreplace',${
+        plugins: [ 'drawio', 'link', 'fullscreen', 'charmap', 'table', 'lists', 'searchreplace',${
```

Comment:
ideally we should make it optional in the configuration fields to have drawio available, with default false

Reply:
is optional now. When you open a row with drawio data but with drawio disabled, you can still see the images, but they are not editable. In the reduced toolbar, the icon is always disabled.